src/status_im/android/platform.cljs

```diff
@@ -37,8 +37,7 @@
                                       :background-color "#7099e619"
                                       :border-radius    5
                                       :padding          4}
-                           :item     {:status-text {:color       styles/color-black
```

Comment:
sorry, but I still don't understand what is the problem there? Could you explain? `line-height` doesn't work when `color` is used, or vise versa? Or what happens?